CONTRIBUTING.md

```diff
@@ -117,6 +117,80 @@ You can learn more about [pull requests on GitHub](https://help.github.com/en/gi
 
 You can also contribute to the project without specifically writing content for the site. Many discussions are happening all around the project. Protocol Labs, and by extension, IPFS, is a collaborative company that relies heavily on the community. We need your opinions and input! If you have an idea about a topic or want to weigh in on a discussion, feel free to have your say either on [GitHub](https://github.com/ipfs/ipfs-docs/issues) or on the [IPFS forums](https://discuss.ipfs.tech/).
 
+## Create a new page
+
+Sometimes you will need to make a whole new page for content to live. The process to do this is outlined below.
+
+These steps were performed in Visual Studio Code (VSCode).
+
+The first step to creating a new page is to identify the type of content it will be to determine where it will live in the structure of the docs. It may need to be a whole new section or it could be added to an existing section as a sub-page. An important concept to keep in mind when deciding where to put the new content is the flow of the already existing content. You wouldn't want to put a page or section designed for experienced users between the already existing **Install** and **Basics** sections.
+
+Once you have determined where the content will live, you must add the appropriate lines to the `config.js` file.
+
+### Create a branch
+
+You can't just add a new page directly to the main branch. First, you will have to create your own branch to push commits to.
+
+To do this in VSCode:
+
+1. Ensure you are in the `ipfs-docs` directory on VSCode.
+1. Click the branch icon in the bottom left of the window.
+    <br><br>
+    <img src="./images/create-new-branch.png" style="display:block;margin:0 auto;width:40vw;">
+    <br>
```

Comment:
Why not us the markdown image format? If you don't like they way markdown images are showing up, you should change how images are deal with in the CSS.

Adding HTML into markdown gets messy really quick.

Reply:
I'll change it to the markdown format. I used HTML to have control over size, but the default size might be okay.

Reply:
You could also just resize the images manually and then re-upload them if you wanted. Depends on what you're trying to resize the image to I guess.

Reply:
Resized the images manually and changed them to markdown image format. Some of them may still be too big in size, I can adjust later if that is the case.